index.ts

```diff
@@ -37,6 +37,36 @@ export class Targetprocess {
         return this.requestJSON(APIVersion.V1, `Userstories/${id}`, "GET");
     }
 
+    private async getEntityState(name: string, processId: number) {
```

Comment:
Shall we make it clearer that this only works for Tasks?  Or perhaps instead add entityTypeName as a parameter.
```suggestion
    private async getTaskEntityState(name: string, processId: number) {
```